### PR TITLE
updates for admin-defined attributes

### PR DESCRIPTION
**Pull request by babbtx**

### CODE REVIEW

File: source/documentation/cloud-apps/api-basics.md [hunk 1]
```diff
@@ -271,17 +271,117 @@ default `per_page` value and may have different limits.  See the specific
...
+}
+```
+
+Name | Type | Description
+-----|------|--------------
+`name`|`string`| Attribute name, i.e. as it would appear in the response object.
+`label`|`string`| Attribute label, e.g. as it should be presented to the user in a form.     
+`type`|`string`| Type of attribute. Values are `string`,`int`,`date`,`enum`,`text`, and `float`.
+`default`|varies| Default value. Can be `null`.
+`options`|array| For `enum` attributes, this is the list of valid values.
```
Comment (msgerbush) [inline source/documentation/cloud-apps/api-basics.md]: should escape `array`

File: source/documentation/cloud-apps/reference/helpdesk.md [hunk 2]
```diff
@@ -141,15 +150,6 @@ Example ticket (note all arrays have been reduced to a single example item):
       "body": "Ticket closed."
     }
   ],
-  "c_alert_level": "orange",
```
Comment (msgerbush) [inline source/documentation/cloud-apps/reference/helpdesk.md]: why not move this into `admin_defined_attrs`?

Comment (babbtx) [inline source/documentation/cloud-apps/reference/helpdesk.md]: I figured it was just copy and paste oversight from the initial docs. None of the examples have any `admin_defined_attrs`. We could add to the examples, but then would that be obvious that it's an example of an admin-defined attribute that will not necessarily be present in the API response? Or would it mislead people to believe it's effectively a "real" attribute always present? Maybe if we provide a ridiculous example like "twix_or_5th_ave_bar"? (I'm all about the Seinfeld references.)

Comment (msgerbush) [inline source/documentation/cloud-apps/reference/helpdesk.md]: ha, i like that example.  good point though, i just was wondering if you'd thought about it, which you clearly have :).

they're all twix

**Pull request merged**